README.md

```diff
@@ -1,36 +1,53 @@
 # OpenCog Hyperon - Distributed Atomspace (DAS)
 
-Here, we keep stuff that is pertinent to the DAS project but doesn't belong to
-any of the other project's specific repositories, such as automated deployment
-scripts, API and high level documentation, assets used by other repositories'
-CI/CD scripts, etc.
+This repository contains items relevant to the DAS project that do not belong in any other project's specific repositories, such as automated deployment scripts, API and high-level documentation, and assets used by other repositories' CI/CD scripts.
```

Comment:
This is no longer true. Just delete it.

Instead, I think we should add a warning like:

ATTENTION: we're currently re-structuring the project repo, documentation etc. So directory structure, documentation files, etc may be outdated or somehow messy.